src/modules/machine/events/store/machineEventsSlice.ts

```diff
@@ -1,29 +1,88 @@
-import type { PayloadAction } from "@reduxjs/toolkit";
-import { createSlice } from "@reduxjs/toolkit";
-import { RootState } from "../../../core/store";
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import { RootState } from '../../../core/store';
+import { machineEventsService } from '../machineEventsService';
+
+const { addMachineEvent, deleteMachineEvent, getList } = machineEventsService;
+
+const useHttpAdd = async (event: Event) => {

```

Comment:
Why do you need a separated method? Please don't use http in naming. It looks odd.